Assets/_BForBoss/_Character/Scripts/DashLinesEffectBehaviour.cs

```diff
@@ -0,0 +1,56 @@
+//-----------------------------------------------------------------------
+// <copyright file="DashLinesEffectBehaviour.cs" company="PERIGON GAMES">
+//     Copyright (c) PERIGON GAMES. 2020 All Rights Reserved
+// </copyright>
+//-----------------------------------------------------------------------
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Perigon.Character
+{
+    public class DashLinesEffectBehaviour : MonoBehaviour
+    {
+        private const string MATERIAL_ALPHA = "_alpha";
+        
+        [SerializeField] [Range(0,1)] float _lerpInStrength = 0.1f;
+        [SerializeField] [Range(0,1)] float _lerpOutStrength = 0.1f;
+        [SerializeField] float _duration = 0.1f;
+        private float _currentTime;
+        
+        [SerializeField] [Range(0,1)] float _maxAlpha = 1f;
+        private Material _speedLineMaterial;
+        
+        private void Start()
+        {
+            _speedLineMaterial = GetComponent<RawImage>().material;
+            _speedLineMaterial.SetFloat(MATERIAL_ALPHA,  0);
+            _currentTime = _duration + 10f;
```

Comment:
Can you also set `10f` as a const up top, and name it appropriately. Since other's probably won't know where this magic 10 comes from.

Reply:
I've renamed it as `START_TIME_OFFSET`, but the idea isn't easy to communicate through the variable name. Basically the _currentTime has to be an amount (arbitrarily 10) above the _duration to ensure that the dash lines start the scene with an alpha of 0 (at the end of the effect). The key point is that _currentTime is GREATER (>) than _duration rather than just being _duration. Thus it is offset by an arbitrary value.